### DISCUSSION TITLE
Wireframes (Landing :flight_arrival: , Explore :mag_right: , User's Profile :open_book:)

### DISCUSSION THREAD

**Discussion by Natashka333** [timmyichen/dev-directory#84] [Ideas]
The Figma link: 
(https://www.figma.com/file/dMGXxsdls6OvlchqcW58Ax/PairUp?node-id=0%3A1&t=k5DaJTnaxRZVuwDO-1)

Here the topics what we need to talk about:
1. Context on the Landing Page;
2. Search/Filter (should they locate next to each other/ are we keeping Search in the nav?);
3. Call/Message;
4. Group events.

Add into comments if i missed anything else.

**Comment by timmyichen** [comment]
Thanks for doing the designs, and for kickstarting this discussion! Breaking my thoughts into a few comments to allow them to be threaded appropriately.

A few of these features are beyond the scope of MVP - namely any kind of messaging/call scheduling, and any kind of group events.  The former is meant to be handled via Discord (without our involvement) and the latter has a bit of overlap with Together and, while an integration would be cool, I don't think either app is ready for such a thing yet.

**Reply by Natashka333** [reply]
Thank you for bringing up your concern. While I envision the schedule a call button taking users to a Calendly link and the message me button leading to Discord, I understand that not everyone may have access to both tools. So, it sounds like mainly Discord, additionally LinkedIn, Calendly (GitHub) links would enable users to connect with each other. Correct?

**Reply by timmyichen** [reply]
Aha, I didn't realize these were meant to be simple links to external services.  Discord is required to use the app so that's guaranteed to be used, but I'm not sure we have the ability to link them into a DM with someone?  That's an interesting idea and definitely worth exploring.  Calendly we can probably do in accordance with your designs. Thanks for clarifying!

**Reply by Caleb-Cohen** [reply]
> Thank you for bringing up your concern. While I envision the schedule a call button taking users to a Calendly link and the message me button leading to Discord, I understand that not everyone may have access to both tools. So, it sounds like mainly Discord, additionally LinkedIn, Calendly (GitHub) links would enable users to connect with each other. Correct?

Schedule a call will be confusing to the user if it's only a link to the calendly. I think links to outside sources like Calendly, or a hyperlink to creating a discord PM should be more explicit on what it is.

**Reply by Natashka333** [reply]
@Caleb-Cohen got you, thanks!

**Comment by timmyichen** [comment]
As a whole (and I've noticed this with the designs for the Remo clone and for Together) there seems to be a big emphasis on page elements being wrapped within bordered containers (usually with rounded borders) - was curious if this was an intentional design choice?  I'm curious if a different style may simplify the UI a bit (and I think we should err on the side of simplicity to start)

**Reply by Natashka333** [reply]
Thank you for bringing up the use of bordered containers in the wireframes. I want to clarify that the containers in this case were used mostly as a visual means of separating and organizing elements within the layout. The final design may differ, and we are definitely open to exploring different styles that would simplify the UI.

**Reply by Caleb-Cohen** [reply]
I like the containerization of some of the content. Specifically, the profile cards in Desktop - 2. That being said, I do agree with @timmyichen that the design could be simplified a bit, especially for MVP.

**Comment by timmyichen** [comment]
For the landing page context, I think for now the only content we need is:

* Main call to action (log in / explore)
* Use cases
* Contributing

**Reply by Natashka333** [reply]
Got it, thank you for clarifying the content needs for the landing page.

**Comment by Caleb-Cohen** [comment]
Is the onboarding page to spec? I thought we were doing one page.
